packages/cloud-cognitive/src/components/PageHeader/PageHeaderUtils.js

```diff
@@ -155,3 +155,12 @@ export const utilCalcSpacingBelowTitle = (
   }
   return belowTitleSpace;
 };
+
+export const utilSetCustomCSSProps = (targetRef, kvPairs) => {
```

Comment:
```suggestion
// Set css style values directly onto a node. Note that this is effective
// immediately, rather than using the React `style` prop which goes through
// the React DOM update optimisation.
export const utilSetCustomCSSProps = (targetRef, kvPairs) => {
```